### PR TITLE
Dark Theme added

### LINKED ISSUE #26: Feat: Implement dark theme for website
- Create a separate folder named `dark-theme` under root folder and implement dark theme there
- Site contents should be clear enough to read and change the font colour of headings, paragraphs etc. if necessary for dark theme
- Create a toggle button and implement logic to enable or disable dark theme on top right corner of navbar
- Creativity is welcome

### PR DESCRIPTION
**Pull request by Anuran12**
Close: #26 

image
image
image

### CODE REVIEW

**Comment by Anuran12** [comment]
Please don't marge this issue before 31 October .

**Comment by thatsuman** [comment]
> Please don't marge this issue before 31 October .

Why?

**Comment by Anuran12** [comment]
Because i am participating in a contest "Hacksquad". there the maintainers are disqualifing teams for dummy project prs. I guess its a dummy project . that's why :(

**Comment by Anuran12** [comment]
@thatsuman 

image

**Comment by Anuran12** [comment]
well, well, you can add this. because one of my dark-mode pr was approved by them. so I hope they will not delete this too. @thatsuman

**Comment by Anuran12** [comment]
Afterall I did hardwork to make this dark-mode too

**Comment by thatsuman** [comment]
Who told you this is a dummy project? Please check out our organization. This repo is a Landing website of our campus management system 'OpenCampus' . We're following every Hacktoberfest rules so that people like you and me can contribute to this project. 
Also we are not accepting any low quality code contribution as we respect contributors hard work.
Please spare me some time so that I can review the changes. 
Thanks!

**Comment by Anuran12** [comment]
@thatsuman Sorry I did not know that... If you want any frontend work on this website.. I would like to work on those... Thanks

**Comment by thatsuman** [comment]
@Anuran12 Please don't be sorry. I'm a developer same as you and in open source community I believe we must help and learn from each other.
Can you please check the toggle button? after enabling dark theme there's no way to disable the dark theme.

**Comment by Anuran12** [comment]
Yea i check that too. I don't know why it's happening. In my local host it's working properly...

File: dark-theme/theme.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var icon = document.getElementById("theme-icon");
+
+icon.onclick = function(){
```
Comment (thatsuman) [inline dark-theme/theme.js]: Can you try with by using arrow function?

**Comment by Anuran12** [comment]
@thatsuman still not working 🥲.

in my localhost its working 

image

**Comment by thatsuman** [comment]
Okay I'm merging this PR and opening another issue for this bug.
Thank you for your contribution!

**Comment by Anuran12** [comment]
hey.. bro why did you merged ?

**Comment by Anuran12** [comment]
I was trying to solving the toggle issue

**Pull request merged**